core/templates/pages/learner-dashboard-page/learner-dashboard-icons.component.spec.ts

```diff
@@ -723,12 +730,12 @@ describe('Learner Dashboard Icons Component', () => {
     const learnerPlaylistSpy =
       spyOn(
         learnerDashboardActivityBackendApiService,
-        'removeFromLearnerPlaylistModal')
-        .and.returnValue(null);
+        'removeFromLearnerPlaylistModal'
+      ).and.callThrough();
 
     component.removeFromLearnerPlaylist(
       activityId, activityTitle, activityType);
-
+    tick();
```

Comment:
Why is this needed?

Reply:
Otherwise, it would throw error(`3 timers still in the queue`) because we've used `callThrough` at 2 other places as well.